cocos/particle/animator/velocity-overtime.ts

```diff
@@ -54,7 +56,7 @@ export default class VelocityOvertimeModule extends ParticleModuleBase {
      * @zh 是否启用。
      */
     @displayOrder(0)
-    public get enable (): boolean {
+    public get enable () {
```

Comment:
why did you remove the return type?

Reply:
actually I didn't Not sure what happened here. prettier?

Reply:
or maybe it was not in the older engine version.